frontend/src/app/[locale]/search/SearchForm.tsx

```diff
@@ -60,26 +60,28 @@ export function SearchForm({
         </div>
         <div className="grid-row grid-gap">
           <div className="tablet:grid-col-4">
-            <SearchOpportunityStatus
-              formRef={formRef}
-              initialQueryParams={statusQueryParams}
-            />
-            <SearchFilterFundingInstrument
-              formRef={formRef}
-              initialQueryParams={fundingInstrumentQueryParams}
-            />
-            <SearchFilterEligibility
-              formRef={formRef}
-              initialQueryParams={eligibilityQueryParams}
-            />
-            <SearchFilterAgency
-              formRef={formRef}
-              initialQueryParams={agencyQueryParams}
-            />
-            <SearchFilterCategory
-              formRef={formRef}
-              initialQueryParams={categoryQueryParams}
-            />
+            <div className="position-sticky top-0 maxh-viewport overflow-auto">
```

Comment:
nit: maybe pull these classes into their own custom style `className="searchFormStickyBar"`

Reply:
We should have a standard and stick to it (either prefer custom class names or not) and document. I don't have a strong opinion either way.

Reply:
Iterated based on that feedback.

Reply:
My opinion is that USWDS already supports custom CSS in the _uswds-theme-custom-styles file and we have already leveraged that for the .pdf-card component. It is cleaner to move large chunks of CSS into a single file to make it easier to read inline.

Reply:
If we add custom classes, they shouldn't duplicate the function of utility classes. USWDS already has a class that makes something sticky, no need to define a new style to do the same thing. I think custom classes should only be used for styles that are not possible with utility classes.